api/v1alpha1/loadbalancer_types.go

```diff
@@ -98,6 +104,11 @@ type ConsistentHash struct {
 	// +optional
 	Cookie *Cookie `json:"cookie,omitempty"`
 
+	// QueryParameter configures the query parameter hash policy when the consistent hash type is set to QueryParameter.
+	//
+	// +optional
+	QueryParameter *QueryParameter `json:"queryParameter,omitempty"`
```

Comment:
lets rename to `queryParam` to keep it consistent with the API https://gateway-api.sigs.k8s.io/reference/spec/#httproutematch

Reply:
Done